torch/optim/adamw.py

```diff
@@ -688,11 +697,11 @@ def _fused_adamw(
     amsgrad: bool,
     beta1: float,
     beta2: float,
-    lr: Union[float, Tensor],
+    lr: Union[Tensor, float],
     weight_decay: float,
     eps: float,
     maximize: bool,
-    capturable: bool,  # Needed for consistency.
```

Comment:
why delete the comment?

Reply:
I thought this is unnecessary because `has_complex` didn't have this comment. But I add it back and add a same comment for `has_complex`.